src/seedu/addressbook/data/AddressBook.java

```diff
@@ -68,6 +68,8 @@ public UniquePersonList getAllPersons() {
         return new UniquePersonList(allPersons);
     }
 
+    public void sortAddressBook() { allPersons.sortUniquePersonList(); }
```

Comment:
According to the [style guide](https://nus-cs2103-ay1819s2.github.io/cs2103-website/coding-standards/java/intermediate.html#layout), it is advisable to "⭐️ Use K&R style brackets (aka Egyptian style)."

```suggestion
    public void sortAddressBook() {
        allPersons.sortUniquePersonList();
    }
```